src/Microsoft.ComponentDetection.Detectors/pip/PipReportComponentDetector.cs

```diff
@@ -81,6 +86,19 @@ protected override async Task OnFileFoundAsync(ProcessRequest processRequest, ID
         FileInfo reportFile = null;
         try
         {
+            if (this.IsPipReportManuallyDisabled())
+            {
+                this.Logger.LogWarning("PipReport: The pip report has been manually disabled, fallback strategy performed.");
```

Comment:
Nit: Comment is a bit off, since there is no fallback strategy, maybe just state that only the standard pip results will be collected?

Reply:
going to just mention that the pip report is skipped, so that in the future when we deprecate existing pip result collection to help with network isolation, we will not need to update the warning message